src/services/transfer/__tests__/transfer-out-ehr-core.test.js

```diff
@@ -198,58 +178,54 @@ describe('transferOutEhrCore', () => {
       Status.SENT_EHR,
       'EHR has been successfully sent'
     );
+    expect(logInfo).toHaveBeenCalledWith('EHR transfer out started');
     expect(logInfo).toHaveBeenCalledWith(`Sending EHR core`);
-    expect(sendCore).toHaveBeenCalledWith(
-      conversationId,
-      odsCode,
-      ehrCoreWithUpdatedMessageId,
-      ehrRequestId,
-      newMessageId
-    );
+    expect(sendCore).toHaveBeenCalledWith(conversationId, odsCode, ehrCoreWithUpdatedMessageId, ehrRequestId, newMessageId);
   });
 
   it('should handle exceptions', async () => {
-    let error = new Error('test error message');
+    const error = new Error('test error message');
     getRegistrationRequestStatusByConversationId.mockRejectedValueOnce(error);
 
-    const result = await transferOutEhrCore({ conversationId, nhsNumber, messageId, odsCode, ehrRequestId });
-    expect(result.hasFailed).toBe(true);
-    expect(result.error).toBe('test error message');
+    await transferOutEhrCore({ conversationId, nhsNumber, messageId, odsCode, ehrRequestId });
     expect(logError).toHaveBeenCalledWith('EHR transfer out request failed', error);
     expect(sendCore).not.toHaveBeenCalled();
   });

```

Comment:
Similar to above can we split this test up with given, when and then comments please? :)

Reply:
Thank you so much!! I have address the PR comments :D